src/main/java/org/xbill/DNS/lookup/AdditionalDetail.java

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: BSD-2-Clause
+package org.xbill.DNS.lookup;
+
+public enum AdditionalDetail {
```

Comment:
- Missing class-level Javadoc
- Needs a SUCESSFUL (or OK) case
- A better name might be `LookupResultDetail`

Reply:
If we decide to go with exceptions instead we get rid of this whole enum